### PR TITLE
[#IOPAE-490] Add Request review legacy queue trigger

### PR DESCRIPTION
**Pull request by giamma1295**
#### List of Changes
- Add `RequestReviewLegacyItem` which maps the request-review-legacy queue item
- Add config for Service Review Legacy Postgres Table
- Add Request Review Legacy Table terraform config
- Add `fetch` method to `fsmLifecycleClient`
- Add `OnRequestReviewLegacy` Azure Function

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
+import { Context } from "@azure/functions";
```
Comment (gunzip) [inline apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts:1]: I'd add a top level comment with the purpose of this module

Comment (giamma1295) [reply apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts:14]: 74b836e

File: apps/io-services-cms-webapp/src/main.ts [hunk 2]
```diff
@@ -140,6 +141,15 @@ export const serviceReviewCheckerEntryPoint = createReviewCheckerHandler(
   fsmLifecycleClient
 );
 
+export const createRequestReviewLegacyEntryPoint =
+  createRequestReviewLegacyHandler(
+    fsmLifecycleClient,
+    getDao({
+      ...config,
+      REVIEWER_DB_TABLE: config.REVIEWER_LEGACY_DB_TABLE,
```
Comment (rodoviti) [inline apps/io-services-cms-webapp/src/main.ts:149]: **suggestion (non-blocking):** by doing as suggested below, we could have avoided modifying infra files and app configs
```suggestion
      REVIEWER_DB_TABLE: `${config.REVIEWER_DB_TABLE}_legacy`,
```

**Review by rodoviti — APPROVED** [review]

File: apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+    queueItem,
+    parseIncomingMessage,
+    TE.fromEither,
+    TE.chain((requestReviewLegacy) =>
+      pipe(
+        requestReviewLegacy.serviceId,
+        fsmLifecycleClient.fetch,
+        TE.chain((service) =>
+          pipe(
+            service,
```
[the remark this answers is not in the document]
Comment (giamma1295) [reply apps/io-services-cms-webapp/src/reviewer/request-review-legacy-handler.ts]: a6a3b5d

File: apps/io-services-cms-webapp/src/main.ts (hunk 2 above)
Comment (giamma1295) [reply apps/io-services-cms-webapp/src/main.ts]: 4a5858e

**Review by giuseppedipinto — APPROVED** [review]

**Pull request merged**